### PR TITLE
Fix xpassed Login events tests for java

### PR DESCRIPTION
**Pull request by jandro996**
## Changes

- Fix tracer version
- Fix tests for some java weblog variants like spring-boot-wildfly

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/appsec/test_automated_login_events.py [hunk 1]
```diff
@@ -1163,6 +1163,8 @@ def validate_login_failure_headers(span):
 
 def assert_priority(span, meta):
```
Comment (jandro996) [inline tests/appsec/test_automated_login_events.py:1164]: We need to check if this assert is needed and if it's correct. As far as I know (at least in java) set the manual.keep tag is not propagated to agent and when is set to true sampling priority is set to 2.

**Review by cbeauchesne — APPROVED** [review]
From framework usage, all good.

But I'll be more confortable if you get a review with somebody confortable with the RFC.

**Review by manuel-alvarez-alvarez — APPROVED** [review]

**Review by cbeauchesne — COMMENTED** [review]
We'll wait for #2810 to re-run the CI

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
You have a CI failure, but it's probably an issue in the weblog implem : 

https://github.com/DataDog/system-tests/actions/runs/10079723533/job/27867983375?pr=2754#step:12:225

**Comment by cbeauchesne** [comment]
I rebased your branch, there were an error on `main` ...

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
Still an error on `spring-boot-openliberty`

https://github.com/DataDog/system-tests/actions/runs/10094392741/job/27912439438?pr=2754#step:12:246

**Comment by jandro996** [comment]
Thanks for your comments @cbeauchesne! as you mentioned, we are having issues with `spring-boot-openliberty` so i won't enable it in this iteration

**Review by daniel-romano-DD — APPROVED** [review]

**Review by smola — APPROVED** [review]

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**